docs/tutorials/hybrid-apps/introduction.md

```diff
@@ -0,0 +1,636 @@
+---
+title: Introduction
+description: Learn how to use FCL with wagmi and rainbowkit to create a hybrid app - one that is simultaneously connected to Flow EVM and Flow Cadence.
```

Comment:
I'm not sure "hybrid app" is clear enough as to what it is. Think we need a better short name.